internal/events/events.go

```diff
@@ -58,6 +59,21 @@ func (m Message) Validate() error {
 	return nil
 }
 
+func NewMessage(ctx context.Context, topic, key, messageType string, data any) (*Message, error) {
```

Comment:
Can you add a godoc to this method explaining that it parses the tenantID from the context and inject it into the message that's returned?

Reply:
PTAL: [ff507b5](https://github.com/stellar/stellar-disbursement-platform-backend/pull/133/commits/ff507b5a378098474562c6c5715d06169a1efc9c)